src/components/Text.js

```diff
@@ -4,13 +4,14 @@ import { Text as RnText, StyleSheet } from "react-native";
 import Markdown from "react-native-easy-markdown";
 import type { Node } from "react";
 import type { StyleObj } from "react-native/Libraries/StyleSheet/StyleSheetTypes";
-import { blackColor } from "../constants/colors";
+import { blackColor, lightNavyBlueColor } from "../constants/colors";
 
 export type TextType = "h1" | "h2" | "h3" | "h4" | "text" | "small" | "price";
 
 type Props = {
   children: Node,
   type?: TextType,
+  color?: "blue",
```

Comment:
I'm not a fan of this style. I think it will encourage the wrong addition of props. Imagine someone else adds a `white` prop. What wins between `blue` and `white`? This leads to impossible states. I think it would be better if we created an `H2` component in the EventDetailsScreen folder for now, setting style and type to the correct setting. I have another idea of refactoring this into just style declarations that we can compose with `Text` components. Personally I think this is a much nicer approach and drastically simplifies components. This can be done later though if we get agreement from the team.

Reply:
Apologies, I thought this was still a `blue` prop.